### ISSUE TITLE
Navigation in Tests

### ISSUE THREAD

**Issue by hakangur** [chamilo/chamilo-lms#2937]
In test options, it would be nice to have a setting that determines the direction of navigation. "Free" navigation means that the student can move back and forth within the test. "Sequential" navigation means that the student can move **_only_** in forward direction and cannot go back to a previous question (this is present in Moodle). This would be very useful for listening activities where students should listen to a passage only once.

**Comment by ywarnier** [comment]
This option has been added in Chamilo 1.11.12 (soon to be released) as a hidden setting. Check for quiz_prevent_backwards_move in main/install/configuration.dist.php, execute the corresponding SQL and add the option as "true" to app/config/configuration.php to enable.
